### ISSUE TITLE
"Start/Resume a User's Playback" is suddenly not working properly 

### ISSUE THREAD

**Issue by gahisee** [spotify/web-api#1434]
Normally I submit an array of URI's to play a temporary playlist.
This has been working for months. But since yesterday, I'm seeing a variety of issues.
1) it plays only the first track and ignores the rest in the submitted list.
2) if an offset is specified, the specified track does play but then as in 1) ignores the rest.
3) it ignores the submission altogether and continues its own (whatever it is) playback.

This is a "dynamic" problem!
My app on both Linux and Android is exhibiting the same issue.
Of course, Spotify app itself behaves fine.

**Comment by joshubrown** [comment]
@gahisee Thanks for the bug report. Can you still reproduce this issue?

**Comment by gahisee** [comment]
Yes. However, the problem only occurs on Bluesound Node device. And I have demonstrated the issue both from my own app as well as using Spotify Console. Furthermore, the problem is fairly new, say within last 4 weeks or so. From what I can discern, the problem is dependent upon the player; i.e., local Spotify app is ok whereas "foreign" device such as Node or Squeezebox has similar problems.

For now, I have found a work-around: replace tracks in a known playlist (e.g., mySpotify) and play it.

**Comment by FrederikRichter** [comment]
I have a problem too, but it is possible that iam just an idiot. I can play an album but not a single track...
It gives me an "unsupportet uri" error
this is my request body https://pastr.io/view/LfgPMZDNJCT
